examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -790,7 +797,12 @@ def __init__(
         self.original_sizes = []
         self.crop_top_lefts = []
         self.pixel_values = []
-        train_resize = transforms.Resize(size, interpolation=transforms.InterpolationMode.BILINEAR)
+
+        if args.image_interpolation_mode == "bilinear":
+            train_resize = transforms.Resize(size, interpolation=transforms.InterpolationMode.BILINEAR)
+        else:
+            train_resize = transforms.Resize(size, interpolation=transforms.InterpolationMode.LANCZOS)
```

Comment:
```suggestion
        interpolation = getattr(transforms.InterpolationMode, args.image_interpolation_mode.upper(), None)
        if interpolation is None:
            raise ValueError(f"Unsupported interpolation mode.")
        train_resize = transforms.Resize(size, interpolation=interpolation)
```

Could we do something like this?

Reply:
yeah, not sure about the other interpolation modes but I guess if the user uses another one it's because they know what they're doing

Reply:
Just cleaner code and preemptively solves the inevitable issue report: "Setting `image_interpolation_mode` other than `bilinear` uses `lanczos`"

Reply:
Let's go with @hlky's suggestion. It's quite nice!